doc/js_libs.pl.md

```diff
@@ -0,0 +1,183 @@
+# Biblioteki JS
+
+# PLGApp API
```

Comment:
Double titles with the same size (`h1`) looks quite ugly:

![double_title](https://cloud.githubusercontent.com/assets/1265430/6801206/1e7f65d2-d226-11e4-94c6-f0a02deecb92.png)
 
Please keep correct hierary (`#`, `##`, `###` converted into `h1`, `h2`, `h3`)